modules/vba_documents/app/mailers/vba_documents/unsuccessful_report_recipients.yml

```diff
@@ -0,0 +1,21 @@
+common:
+  - 'michael.bastos@oddball.io'
+  - 'lydia.vian@thunderyard.com'
+  - 'joshua.jennings@libertyits.com'
+  - 'cristopher.shupp@libertyits.com'
+  - 'gregory.bowman@libertyits.com'
+  - 'mark.viterna@libertyits.com'
+dev:
+  - 'dev@dev.dev'
+staging:
+  - 'staging@staging.staging'
+sandbox:
+  - 'sandbox@sandbox.sandbox'
```

Comment:
Are these emails configured somewhere else in the system? They look like placeholders at first glance.

Reply:
They are placeholders.  Someone from another team was asked to look at those placeholders and tell us if any members of his team wanted to be placed in those spots.

Reply:
Oh and yes I need to blow them away before the squash :-)

Reply:
Could we clean these up before review?